crud/src/main/java/com/redhat/lightblue/rest/crud/health/CrudCheckConfiguration.java

```diff
@@ -21,18 +13,28 @@
 import com.redhat.lightblue.rest.auth.ldap.LdapRolesProvider;
 import com.redhat.lightblue.rest.health.ControllerHealthCheck;
 import com.redhat.lightblue.util.JsonUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-/**
- * Listener to register Health Check classes to Metrics Health check registry
- */
-public class CrudCertAuthHealthCheckServletContextListener extends HealthCheckServlet.ContextListener {
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class CrudCheckConfiguration {
 
-    private final Logger LOGGER = LoggerFactory.getLogger(CrudCertAuthHealthCheckServletContextListener.class);
-    
-    @Override
-    protected HealthCheckRegistry getHealthCheckRegistry() {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CrudCheckConfiguration.class);
 
+    public static HealthCheckRegistry getHealthCheckRegistry() {
```

Comment:
Maybe `newHealthCheckRegistry()` or `createHealthCheckRegistry()` makes it clear this doesn't return a singleton.